### PR TITLE
[14.0][ADD] add module to remove app download links

### PR DESCRIPTION
**Pull request by PicchiSeba**
Proposal for a module to disable the Odoo Enterprise app download window when no expenses have been registered

Before:
Screenshot from 2023-03-27 15-54-24

After:
Screenshot from 2023-03-27 15-52-48

I think this could be an interesting addition to the de-brand options.
This module only depends on the `hr_expense` module, which is also the reason why it wasn't included in previously available de-brand modules.

### CODE REVIEW

**Comment by PicchiSeba** [comment]
pre-commit error doesn't seem to be related to my changes, can anyone confirm?

File: remove_mobile_app_links/views/hr_expense.xml
Comment (astirpe) [inline remove_mobile_app_links/views/hr_expense.xml]: Here you fixed the view `hr_expense_actions_my_unsubmitted` but the same fix should be done also for `hr_expense_actions_my_all`

File: remove_mobile_app_links/__manifest__.py
Comment (astirpe) [inline remove_mobile_app_links/__manifest__.py]: The developer names are already included in `CONTRIBUTORS.rst`, so remove them from the `"author"` and only keep the company name.

**Review by astirpe — CHANGES_REQUESTED** [review]
Seems to me that the pre-commit failure is unrelated to this PR

File: remove_mobile_app_links/__manifest__.py
Comment (HviorForgeFlow) [reply remove_mobile_app_links/__manifest__.py]: Also when pre-commit will work it will fail because `Odoo Community Association (OCA)`is missing in author value

**Comment by PicchiSeba** [comment]
I applied the suggested fixes but the error seems to be the same as before. I guess we'll have to wait.

**Review by astirpe — APPROVED** [review]
Code review is ok for me.
I don't know if the module should renamed in order to mention that it extends hr_expense (something like `remove_expense_mobile_app_link`, but I would let someone else discuss/decide.
So I approve the module as it is.

**Comment by HekkiMelody** [comment]
I agree with the rename to `hr_expense_remove_mobile_link`

Otherwise, looks good!

Will approve after the name change.

**Review by HekkiMelody — APPROVED** [review]

**Comment by HekkiMelody** [comment]
@OCA/brand-maintainers

**Comment by astirpe** [comment]
@PicchiSeba will you also port this module to V15 and V16?

**Comment by PicchiSeba** [comment]
@astirpe
Yes

**Pull request merged**
